esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
Had to remove that to get the IDF to compile, since some functions are hard-coded to use 2 cores.  I did test it, and it's working fine.  If you're interested in the exact area, I can try compiling with `CONFIG_FREERTOS_UNICORE ` restored to `1`.

Depends on what you mean by "test," hence my question in #88 WRT running tests.

  I was just in "make it work" mode.

I did a `make menuconfig` on one of the IDF examples, and it un-defines this.